backend/src/routes/question.js

```diff
@@ -1,5 +1,5 @@
 const express = require('express')
-const router = express.Router({ mergeParams: true })
```

Comment:
no longed needed?

Reply:
Yeah. Originally we had a compound key for questions and it was kind of hacky. Since everything now has a unqiue ID you don't necessarily need the course ID to access question data. The course ID can still be sent along with POST's ect.

Reply:
Yeah cause we now get questions at /api/question rather than api/course/:id/question, with question ids being guaranteed to be unique across all courses.